pedalboard_native/io/__init__.pyi

```diff
@@ -319,7 +319,9 @@ class ReadableAudioFile(AudioFile):
         """
         Close this file, rendering this object unusable.
         """
-    def read(self, num_frames: int = 0) -> numpy.ndarray[typing.Any, numpy.dtype[numpy.float32]]:
+    def read(
+        self, num_frames: typing.Union[float, int] = 0
```

Comment:
I'm a little worried about this change. What does it mean for pedalboard to return a fractional sample?